test/functional/FunctionalCodeAction.hs

```diff
@@ -513,8 +513,8 @@ missingPragmaTests = testGroup "missing pragma warning code actions" [
             let expected =
 -- TODO: Why CPP???
 #if __GLASGOW_HASKELL__ < 810
-                    [ "{-# LANGUAGE ScopedTypeVariables #-}"
-                    , "{-# LANGUAGE TypeApplications #-}"
+                    [ "{-# LANGUAGE TypeApplications #-}"
+                    , "{-# LANGUAGE ScopedTypeVariables #-}"
 #else
                     [ "{-# LANGUAGE TypeApplications #-}"
                     , "{-# LANGUAGE ScopedTypeVariables #-}"
```

Comment:
This macro seems to be redundant now

Reply:
Thanks, I missed this but since adding pragmas to the end of the list of pragmas, it is necessary again anyway.